ccm_web/server/appHandler.ts

```diff
@@ -41,15 +42,9 @@ class AppHandler {
       { host: db.host, dialect: 'postgres', logging: false }
     )
 
-    let devMode
     let staticPath
     const cookieOptions = { secure: true, sameSite: 'None' }
-    if (this.envOptions.isDev) {
-      // Setting devMode to true because cookies seem to be lost during proxy?
-      devMode = true
```

Comment:
This was not needed anymore so now local configuration was acting as Prod mode. So probably session validation is happening in local as well